chapters/2.md

```diff
@@ -204,7 +208,7 @@ To avoid loss due to theft, never share your seed or keys with anybody else. Any
 
 When sending any large transaction to a new person or exchange, you should always initially test the address/service by sending a smaller amount first. When sending any large transaction to a new person or address, you should always send a tiny pilot transaction first and wait for the recipient to confirm that it arrived. This is an important habit to catch mistakes in advance, since there is no “undo”  button for cryptocurrencies.
```

Comment:
Here the first and second sentences look very similar. I understand their differences though.
Maybe rephrase one of them?